commands/account/rename.ts

```diff
@@ -63,3 +60,23 @@ export function builder(yargs: Argv): Argv<AccountRenameArgs> {
 
   return yargs as Argv<AccountRenameArgs>;
 }
+
+const builder = makeYargsBuilder<AccountRenameArgs>(
+  accountRenameBuilder,
+  command,
+  describe,
+  {
+    useGlobalOptions: true,
```

Comment:
Non-blocking (and doesn't make sense to be part of this PR) but we may just want to update `useGlobalOptions` to default to true. Not sure if there are any commands where it's false

Reply:
Yeah that's a good idea. Agreed 👍